cmd/osde2ectl/get/cmd.go

```diff
@@ -65,10 +78,32 @@ func init() {
 		false,
 		"A flag that triggers the fetching of a given cluster's kubeconfig.",
 	)
+	pfs.Uint64VarP(
+		&args.hours,
+		"hours",
+		"r",
+		0,
+		"Cluster expiration extension value in hours.",
+	)
+
+	pfs.Uint64VarP(
+		&args.minutes,
+		"minutes",
+		"m",
+		0,
+		"Cluster expiration extension value in minutes.",
+	)
+
+	pfs.Uint64VarP(
+		&args.seconds,
+		"seconds",
+		"s",
+		0,
+		"Cluster expiration extension value in seconds.",
+	)
+
 	viper.BindPFlag(config.Cluster.ID, Cmd.PersistentFlags().Lookup("cluster-id"))
-	Cmd.RegisterFlagCompletionFunc("output-format", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
-		return []string{"json", "prom"}, cobra.ShellCompDirectiveDefault
-	})
+	viper.BindPFlag(config.Kubeconfig.Path, Cmd.PersistentFlags().Lookup("kube-config-path"))
```

Comment:
Thinking on this, why don't we get rid of the viper.Binds entirely? We should be able to do most of osde2ectl without using viper.